app/views/headerFooterTemplate/Layout.scala.html

```diff
@@ -32,9 +34,18 @@
         taxYear: Option[Int] =  None
 )(contentBlock: Html)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
+@headerBlock = {
+    @govukHeader(Header(
+        homepageUrl = Some("https://www.gov.uk/"),
+        serviceName = Some(messages("service.name")),
+        serviceUrl =  Some(messages("service.homePageUrl", taxYear.getOrElse(2020).toString))
```

Comment:
I think it would be good to refactor this to call the controller route directly as it would work across all environments (where the url differs per environment E.g. qa.tax.service)

I also think it would be good to set the defaultTaxYear in config so we have control over it. 

So in FrontendAppConfig & applicationConfig something like 
`appConfig.defaultTaxYear = 2022`

And then homepageUrl & serviceUrl would become
`Some(controllers.routes.StartPageController.show(appConfig.defaultTaxYear).url)`